README.md

```diff
@@ -1,19 +1,43 @@
-# graft-ng
-//////////////////////////////////////////
-//CMake install
-https://cmake.org/download/
+# Graft Supernode (graft-ng)
 
-sudo /bin/sh cmake-3.11.1-Linux-x86_64.sh --prefix=/opt/cmake --skip-license
+## Compiling Graft Supernode from Source
+
+### Dependencies
+
+| Dependency     | Min. Version  | Debian/Ubuntu Pkg  | Arch Pkg       | Optional | Purpose          |
+| -------------- | ------------- | ------------------ | -------------- | -------- | ---------------- |
+| GCC            | 4.7.3         | `build-essential`  | `base-devel`   | NO       |                  |
+| CMake          | 3.11.0        | `cmake`^           | `cmake`        | NO       |                  |
+| pkg-config     | any           | `pkg-config`       | `base-devel`   | NO       |                  |
+| Boost          | 1.58          | `libboost-all-dev` | `boost`        | NO       | C++ libraries    |
+| OpenSSL        | basically any | `libssl-dev`       | `openssl`      | NO       | sha256 sum       |
+| autoconf       | any           | `autoconf`         |                | NO       | libr3 dependency |
+| automake       | any           | `automake`         |                | NO       | libr3 dependency |
+| check          | any           | `check`            |                | NO       | libr3 dependency |
+| PCRE3          | any           | `libpcre3`         |                | NO       | libr3 dependency |
```

Comment:
are you sure we need both PCRE3 and PCRE3-devel? Usually *-devel should be enough

Reply:
You are right, removed.